src/grasshopper/lib/util/shapes.py

```diff
@@ -196,3 +197,52 @@ def __init__(self, *args, **kwargs):
             },
         ]
         super().__init__(*args, **kwargs)
+
+class Customstages(Default):  # noqa E501
+    """
+    Stolen from this set of examples as part of the locust.io documentation.
```

Comment:
nit: you don't really need this comment, it is relevant to the `Stages` class

Reply:
Removed